src/ethereum_test_tools/common/types.py

```diff
@@ -2616,6 +2640,10 @@ class FixtureBlock:
     Representation of an Ethereum block within a test Fixture.
     """
 
+    @staticmethod
+    def _header_encoder(header: Header) -> FixtureHeader:
+        return FixtureHeader.from_header(header)
+
```

Comment:
These custom encoders simply use `load_dataclass_from_json` for the case where the input type is a dictionary.

Ideally we remove these additions from `FixtureBlock` and create entirely new seperate encoders for `FixtureEngineNewPayload` or rather a new type `ConsumeEngineNewPayload`. This would mean adding a `types/consume/` folder in the future where `ConsumeEngineNewPayload` would reside.

Hopefully at that point we can remove `FixtureEngineNewPayload` completely, as well as the hive blockchain test fixtures. But lets get invalid tests, logging and debugging finessed first!